### PR TITLE
Green was a little too bright.

### PR DESCRIPTION
**Pull request by marjohn56**
Reduced green by around 15%

### CODE REVIEW

**Comment by fichtner** [comment]
Revision or version bump please, no leaking back the source map either... it had issues with Chrome in the past :(

File: misc/theme-rebellion/src/opnsense/www/themes/rebellion/build/css/main.css [hunk 1]
```diff
@@ -5846,3 +5846,5 @@ select {
   background-repeat: no-repeat;
   background-position: right;
   background-image: url(/ui/themes/opnsense/build/images/caret.png) !important; }
+
+/*# sourceMappingURL=main.css.map */
```
Comment (fichtner) [inline misc/theme-rebellion/src/opnsense/www/themes/rebellion/build/css/main.css]: These two lines also gone, belong to source map we really don't need.

**Comment by marjohn56** [comment]
Done

**Comment by fichtner** [comment]
Merged, thanks ❤️

**Pull request merged**